spec/support/sidekiq_unique_jobs/testing.rb

```diff
@@ -59,12 +59,12 @@ def debug(*args)
         redis { |conn| conn.debug(*args) }
       end
 
-      def flushall(options = nil)
-        redis { |conn| conn.flushall(options) }
+      def flushall(...)
+        redis { |conn| conn.flushall(...) }
       end
 
-      def flushdb(options = nil)
-        redis { |conn| conn.flushdb(options) }
+      def flushdb(...)
```

Comment:
This and `flushall` was broken for the default case, i.e. where `nil` was provided.